.github/workflows/pytest.yml

```diff
@@ -30,9 +30,15 @@ jobs:
         run: |
           curl -LsSf https://astral.sh/uv/install.sh | sh
           uv pip install --system coverage coveralls
-      - name: Install package
+      - name: Install pynx without nomad plugin
+        if: "${{ matrix.python_version == '3.8' }}"
```

Comment:
We could also drop support for py3.8. but it's fine for now.

Reply:
Yes, I would only drop it when it's end of life or we have some explicit code in pynx which renders it incompatible. I was hesitant to generally drop it because of it being a plugin.